test/command/test-base.rb

```diff
@@ -203,14 +203,88 @@ def test_curly_brackets_format
         VALUES
 
         expected = <<-OUTPUT
-{"index":{"_index":"groonga","_type":"Site"}}
+{"index":{"_index":"site","_type":"groonga"}}
 {"_key":"http://example.org/","title":"This is test record 1!"}
 {"_key":"http://example.net/","title":"This is test record 2!"}
         OUTPUT
 
         assert_equal(expected.chomp, load.to_elasticsearch_format)
       end
     end
+
+    sub_test_case("options") do
+      def test_options_elasticsearch_version_5
```

Comment:
Can we use parameterized test?

ref: https://www.clear-code.com/blog/2013/1/23.html